test/extensions/common/wasm/test_data/test_cpp/toolchain/BUILD

```diff
@@ -0,0 +1,46 @@
+licenses(["notice"])  # Apache 2
```

Comment:
Could we add this toolchain to the SDK? It's going to be PITA if every extension needs to maintain their own copy of it.

Reply:
I have no idea if that is possible.  Is it possible to pull a toolchain from a remote repo?

Reply:
I think it is possible: https://github.com/bianpengyuan/wasm-example/blob/c81145a7e579ade2def5515c6e70a19623fd70a1/.bazelrc#L17

Reply:
Done.